blog/models.py

```diff
@@ -27,6 +28,10 @@ class Meta:
     def __str__(self):
         return smart_text(self.title)
 
+    @property
+    def description_html(self):
```

Comment:
Buna markdownx markdown ve HTML versiyonunu iki ayrı field'da tutmadığından ve bizdeki eski kayıtlar varsayılan olarak Markdown formatında olduğu için mi ihtiyaç var? Bu methoddan kurtulmak için bir de data migration mı gerekecek?

Eğer öyleyse, daha düzgün bir proje yok mu kullanacak? :)

Reply:
Onceki plugin rendered halini farklı field üzerinde tutuyordu, markdownx kendisi `markdownify` (slugify gibi) bir tool veriyor. Bu property olduğu için migration-da yok zaten, migration eski ve artık kullanılmayan field-i siliyor, yeni field eklemiyor. Normalde bu `templatetag` olarak verilmeliydi bence, ama utils olarak verilmiş. Property-i kaldırıp bir template tag de ekleye bilirim blog-a.

Admin kısmında `MarkdownxModelAdmin` kullandığımız için bunu kendisi yapıp render ediyor. Aslında eski yöntem biraz gereksiz olmuş gibi.